tox.ini

```diff
@@ -79,6 +79,7 @@ omit =
   salt/_modules/metalk8s_package_manager_apt.py
 
 [testenv:unit-tests]
+basepython = python3.6
```

Comment:
I'm surprised about this, wasn't it included in #3163? [relevant commit](https://github.com/scality/metalk8s/pull/3163/commits/3c736a0df7a4294609f75d94b392a6ec3a775000)

Reply:
Anyway it will not be in the resulting commit in the queue so it's ok